tests/Dockerfile

```diff
@@ -6,7 +6,9 @@ RUN apt-get update && apt-get install -y ruby apache2 vim curl ruby-dev build-es
     && echo "deb http://debian.datastax.com/community stable main" | tee -a /etc/apt/sources.list.d/cassandra.sources.list \
     && curl -L http://debian.datastax.com/debian/repo_key | apt-key add - \
     && apt-get update && apt-get install -y cassandra \
-    && gem install --no-ri --no-rdoc cassandra-driver \
+    && gem install rubygems-update \
+    && update_rubygems \
```

Comment:
Its because the latest cassandra-driver requires ruby >= 2.2.0. That said I dont' think the PR would work even as is, since it leaves the cassandra-driver uninstalled.